apps/lead/models.py

```diff
@@ -277,6 +277,13 @@ def get_authors_display(self):
             return self.author and self.author.data.title
         return self.author_raw
 
+    def get_authoring_organizations_type_display(self):
+        if self.authors.exists():
+            return ','.join([author.get_organization_type_display() for author in self.authors.all()])
```

Comment:
We need to use `set` here, multiple authors can have the same organization type.

```suggestion
            return ','.join(set([
                author.get_organization_type_display() for author in self.authors.all()
            ]))
```